group.lisp

```diff
@@ -101,6 +101,14 @@ needs to redraw anything on it, this is where it should do it."))
   (:documentation "When a head or its usable area is resized, this is
 called. When the modeline size changes, this is called."))
 
+(defmethod group-current-window (group)
+  (screen-focus (group-screen group)))
```

Comment:
Ah, thanks for catching this! I think the best way to address this would be to keep track of the current window on the group object itself. What do you think?